vpr/src/server/telegramparser.cpp

```diff
@@ -0,0 +1,71 @@
+#ifndef NO_SERVER
+
+#include "telegramparser.h"
+#include "convertutils.h"
+#include "commconstants.h"
+
+namespace comm {
+
+std::optional<std::string> TelegramParser::try_extract_json_value_str(const std::string& json_string, const std::string& key) {
+    static const std::string end_key_pattern{"\":\""};
```

Comment:
As prior comments, I wouldn't use the `static` keyword here. I would use either constexpr or leave it as is.

Reply:
fixed https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2526/commits/e1b578a682a7e2b6c2a26178187798f16ae83d49